Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tomcat:9.0.37-jdk8-openjdk-slim
+FROM tomcat:9.0.48-jdk11-openjdk-slim
```

Comment:
It would be good to note the motivation for this change in the commit message - that the current version of UAA needs JDK11 to build

Reply:
Updated commit message.